packages/uni_app/lib/view/home/edit_home.dart

```diff
@@ -14,29 +17,93 @@ class EditHomeViewState extends State<EditHomeView> {
   List<DraggableTile> activeCards = [];
   List<DraggableSquare> listlessCards = [];
 
+  (String, Icon) formatDraggableTile(FavoriteWidgetType2 favorite) {
+    String title;
+    Icon icon;
+
+    switch (favorite.name) {
+      case 'schedule':
+        title = 'Schedule';
+        icon = const Icon(Icons.schedule);
```

Comment:
Don't forget to use phosphor icons by adding them to UniIcons in uni_ui and import them from there